config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -0,0 +1,90 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-storage-kind-disruptive
+    always_run: false
```

Comment:
I recommend `cluster: eks-prow-build-cluster` instead of the old google internal default cluster we're trying to phase out, but that can be a follow up. (we also have a k8s-infra GKE cluster but we're trying to adopt the new amazon credits this year and figure out what jobs remain that _have_ to use GCP instead of being flexible between either, there should be a PSA soon from xmundrii)